extensions/wikia/UserLogin/UserLoginSpecialController.class.php

```diff
@@ -415,7 +425,15 @@ public function login() {
 		switch ( $loginCase ) {
 			case LoginForm::SUCCESS:
 				// first check if user has confirmed email after sign up
-				if (  $this->wg->User->getOption( self::NOT_CONFIRMED_SIGNUP_OPTION_NAME ) == true ) {
+				if ( $this->wg->User->getOption( self::NOT_CONFIRMED_SIGNUP_OPTION_NAME ) == true &&
```

Comment:
If this line has a weak comparison with true, could '== true' part be removed or the comparison stronger?